### DISCUSSION TITLE
[Feature] Support selecting / multi-selecting images in Folder view

### DISCUSSION THREAD

**Discussion by canton7** [immich-app/immich#12480] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

The new folder view is really good, thanks for implementing it!

It would be perfect if we could select images from the folder view, and then add that selection to an album.

My use-case is that my assets are stored in an external library. When I add new assets (which are added to a new folder), I want to be able to easily select them all and add them to a new album). However, the only way to do this currently is using the API.

Being able to select images from the folder view, in the same way as from the timeline, and then add that selection to an album, would really help.

Thanks!

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#9159

**Reply by nielstron** [reply]
now reopened in #14307
